src/frontend/efiling-demo/src/components/page/home/Home.js

```diff
@@ -6,7 +6,8 @@ import { Button } from "../../base/button/Button";
 
 import "../page.css";
 
-const generateUrlBody = {
+const urlBodyWithCSO = {
+  userId: "77da92db-0791-491e-8c58-1a969e67d2fa",
```

Comment:
do we care about storing this in the repo?

Reply:
it was just for temporary, since this id isn't private I dont think there is any harm storing it directly in code for now. Eventually this wont be part of our app anyways since it is the client